modules/apps/layout/layout-admin-web/src/main/java/com/liferay/layout/admin/web/internal/display/context/MillerColumnsDisplayContext.java

```diff
@@ -540,6 +635,17 @@ else if (layout.isTypeContent()) {
 			}
 		}
 
+		if (_ffLayoutTranslationConfiguration.enabled()) {
```

Comment:
Oh, I forgot to mention that you need to include a "url" field in the JSON object for the dropdown to enable itself. Any value will do (we don't use it).